Cogs/Image.py

```diff
@@ -79,6 +81,92 @@ async def flip(self, ctx, mode='n'):
         os.remove(img_name)
         os.remove(tmp_img_name)
 
+    @commands.command()
+    async def alpha(self, ctx, mode='n'):
+        
+        # 回数制限があるAPI用のcountを持つjson
+        api_count = 'api_count.json'
+        with open(api_count, encoding="utf8") as f:
+            api_count_json = json.loads(f.read())
+        
+        # 制限回数を更新
+        dt_now = datetime.datetime.now()
+        if api_count_json['alpha']['year'] != dt_now.year or api_count_json['alpha']['month'] != dt_now.month:
+             api_count_json['alpha']['count'] = 50
+             api_count_json['alpha']['year'] = dt_now.year
+             api_count_json['alpha']['month'] = dt_now.month
+
+        # 50回制限確認
+        if api_count_json['alpha']['count'] <= 0:
+            await ctx.send('月の50回越えてそうだからやめておこう')
+            return
+
+        try:
+            # アップロードした画像URL、画像名取得
+            img_url = ctx.message.attachments[0].url
+            file_name = ctx.message.attachments[0].filename
+        except:  # noqa
+            await ctx.send('画像が足りないよ？')
+            return
+        
+        # urlを元に画像を取得し、tmpフォルダに保存
+        tmp_img_name = 'temp/{0}'.format(file_name)
+        
+        # ディレクトリ作成(存在していれば作成しない)
+        os.makedirs('temp/', exist_ok=True)
 
+        r = requests.get(img_url)
+        if r.status_code == 200:
+            with open(tmp_img_name, 'wb') as f:
+                f.write(r.content)
+
+        # 背景透過のAPIURL
+        url = "https://api.remove.bg/v1.0/removebg"
+
+        # post実行
+        result = requests.post(url,
+            data = {'size' : 'auto'},
+            headers={'X-API-Key': 'LrTEjBryRmFCUgJWkx76G5Yd'},
+            files={'image_file' : open(tmp_img_name, 'rb')})
+
+        # status_codeを代入
+        status_code = result.status_code
+
+        # 成功
+        if status_code == requests.codes.ok:
+            img_name = 'temp/alpha/{0}'.format(file_name)
+
+            # ディレクトリ作成(存在していれば作成しない)
+            os.makedirs('temp/alpha/', exist_ok=True)
+
+            with open(img_name, 'wb') as out:
+                out.write(result.content)
+                
+            # 透過後の画像サイズ取得し、確認
+            flip_size = os.path.getsize(img_name)
+            if flip_size > 8178892:
+                await ctx.send('8M超えました。')
+                os.remove(img_name)
+                os.remove(tmp_img_name)
+                return
+            
+            # ファイルを取得し返す
+            with open(img_name, "rb") as fh:
+                f = discord.File(fh, filename=file_name)
```

Comment:
ここのfhも同じく、こんな感じにするとよさそう
```suggestion
                f = discord.File(io.BytesIO(r.content), filename=file_name)
```